src/DashboardView/Settings/EventTypes/EventTypes.js

```diff
@@ -52,9 +53,15 @@ class EventTypes extends Component {
         this.fetchEventTypes();
     }
 
-    delteEventType(eventType) {
+    deleteEventType(eventType) {
```

Comment:
Please rename this method to `handleDeleteEventType`, because it's the base method of a callback.